documentation/specs/Button.md

```diff
@@ -13,31 +13,31 @@ export type ButtonVariant = "filled" | "outlined" | "link";
 export type ButtonSize = "sm" | "md";
 export type ButtonProps = {
   /** Button color */
-  // Defaults to primary
+  /** @default "primary" */
```

Comment:
**note:** just an fyi that TSDoc/JSDoc should be a single comment block and not split up across blocks. for example, before:

```tsx
/** Button color */
/** @default "primary" */
color?: ButtonColor;
```
this will only show `@default "primary"` in IDE intellisense.

after:

```tsx
/**
 * Button color
 * @default "primary"
 */
color?: ButtonColor;
```

this will properly show both the summary and `@default` in intellisense